Source/Core/Common/StringUtil.cpp

```diff
@@ -230,6 +230,16 @@ std::string StringFromInt(int value)
 	return temp;
 }
 
+std::string StringFromFloat(float value)
+{
+	return StringFromFormat("%.9g", value);
+}
+
+std::string StringFromDouble(double value)
+{
+	return StringFromFormat("%.17g", value);
+}
```

Comment:
What is the purpose of using these when StringFromFormat can just be used directly within IniFile now?